tox.ini

```diff
@@ -24,11 +24,12 @@ basepython =
 deps =
     dj18: Django>=1.8.1,<1.9
     dj110: Django>=1.10,<1.11
-    dj111: Django>=1.11,<1.12
+    dj111: Django>=1.11,<2.0
```

Comment:
Why not keep 1.12? I think I've understood Django won't have a 1.12, but it still seems more correct in that case.